pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_seed.go

```diff
@@ -98,6 +98,9 @@ func (g *graph) handleSeedCreateOrUpdate(seed *gardencorev1beta1.Seed) {
 	namespaceVertex := g.getOrCreateVertex(VertexTypeNamespace, "", gutil.ComputeGardenNamespace(seed.Name))
 	g.addEdge(namespaceVertex, seedVertex)
 
+	leaseVertex := g.getOrCreateVertex(VertexTypeLease, gardencorev1beta1.GardenerSeedLeaseNamespace, seed.Name)
```

Comment:
I'm wondering why above in line 95 we delete all incoming edges for `VertexTypeNamespace` but don't need to do it for `VertexTypeLease`. Can you shed some light on it?

Reply:
Thanks, fixed with 36e47ab :)